### PR TITLE
update schema with latest from xtUML metamodel

### PR DESCRIPTION
**Pull request by leviathan747**
I updated the schema as you suggested and confirmed that it worked on recent versions of MicrowaveOven and GPS Watch (I built and ran the application with my model compiler).

### CODE REVIEW

File: bridgepoint/schema.py [hunk 1]
```diff
@@ -2245,15 +2307,15 @@
 CREATE ROP REF_ID R501 FROM MC SM_STATE (SM_ID) TO 1 SM_SM (SM_ID);
 CREATE ROP REF_ID R502 FROM MC SM_EVT (SM_ID) TO 1 SM_SM (SM_ID);
 CREATE ROP REF_ID R503 FROM MC SM_SEME (SM_ID, SMstt_ID) TO 1 SM_STATE (SM_ID, SMstt_ID);
-CREATE ROP REF_ID R503 FROM MC SM_SEME (SM_ID, SMevt_ID) TO 1 SM_SEVT (SM_ID, SMevt_ID);
-CREATE ROP REF_ID R504 FROM 1C SM_EIGN (SM_ID, SMevt_ID, SMstt_ID) TO 1 SM_SEME (SM_ID, SMevt_ID, SMstt_ID);
-CREATE ROP REF_ID R504 FROM 1C SM_CH (SM_ID, SMevt_ID, SMstt_ID) TO 1 SM_SEME (SM_ID, SMevt_ID, SMstt_ID);
-CREATE ROP REF_ID R504 FROM 1C SM_NSTXN (SM_ID, SMevt_ID, SMstt_ID) TO 1 SM_SEME (SM_ID, SMevt_ID, SMstt_ID);
+CREATE ROP REF_ID R503 FROM MC SM_SEME (SMevt_ID, SM_ID) TO 1 SM_SEVT (SMevt_ID, SM_ID);
```
Comment (john-tornblom) [inline bridgepoint/schema.py:2310]: Hmmm, have the order of these identifiers changed? I've been out of the loop for a while so not sure what changed in the metamodel

Comment (leviathan747) [reply bridgepoint/schema.py:2310]: not sure... i would not have expected the order to change. 
image

Comment (john-tornblom) [reply bridgepoint/schema.py:2310]: Actually, now when I think about it, I believe the order is irrelevant here.
Links are created by batch-relate, which uses attribute names when searching for instances.
Sorry for the false alarm :)

File: bridgepoint/schema.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 # License along with pyxtuml. If not, see <http://www.gnu.org/licenses/>.
 
 
-__version__ = 5.2
+__version__ = 5.3
```
Comment (john-tornblom) [inline bridgepoint/schema.py]: The version number I picked previously was based on the bridgepoint editor version from which the schema was first introduced (i.e. currently bridgepoint 5.2). I would bump this to the current version

Comment (leviathan747) [reply bridgepoint/schema.py]: done

**Comment by john-tornblom** [comment]
great, thanks!

**Pull request merged**